crates/git/src/hosting_provider.rs

```diff
@@ -71,24 +71,32 @@ impl HostingProvider {
         commit: Oid,
         client: Arc<dyn HttpClient>,
     ) -> Result<Option<Url>> {
-        match self {
+        let (author, query) = match self {
             HostingProvider::Github => {
                 let commit = commit.to_string();
+                (
+                    github::fetch_github_commit_author(repo_owner, repo, &commit, &client).await?,
+                    Some("size=128"),
+                )
+            }
+            HostingProvider::Codeberg => {
+                let commit = commit.to_string();
+                (
+                    codeberg::fetch_codeberg_commit_author(repo_owner, repo, &commit, &client)
+                        .await?,
+                    None,
+                )
+            }
+            _ => (None, None),
```

Comment:
This seems overly complicated: when only 1/3 cases uses the URL query, I think we should set the size inside the `HostingProvider::Github` arm.

Reply:
Okay, done